src/vendors/oasisscan.ts

```diff
@@ -103,3 +121,25 @@ export function parseTransactionsList(transactionsList: OperationsRow[]): Transa
     return parsed
   })
 }
+
+export function parseDelegations(delegations: DelegationRow[]): Delegation[] {
+  return delegations.map(delegation => {
+    const parsed: Delegation = {
+      amount: parseStringValueToInt(delegation.amount).toString(),
+      shares: parseStringValueToInt(delegation.shares).toString(),
+      validatorAddress: delegation.validator_address,
+    }
+    return parsed
+  })
+}
+export function parseDebonding(debonding: DebondingDelegationRow[]): DebondingDelegation[] {
+  return debonding.map(debonding => {
+    const parsed: DebondingDelegation = {
+      amount: (parseStringValueToInt(debonding.shares) * 1).toString(),
```

Comment:
When debonding, are shares always equal to amount?
[Debonding API] only returns shares, unlike [delegations API]

[Debonding API]: https://api.oasisscan.com/mainnet/chain/account/debonding?address=oasis1qzah5wn48ekakmq5405qvcg4czp8hjvrcvcywvhp&size=500
[delegations API]: https://api.oasisscan.com/mainnet/chain/account/delegations?address=oasis1qzah5wn48ekakmq5405qvcg4czp8hjvrcvcywvhp&size=500

Reply:
related code in oasisscan:
- [delegations calculating amount](https://github.com/bitcat365/oasisscan-backend/blob/f1dafe3/src/main/java/romever/scan/oasisscan/service/AccountService.java#L246-L251)
- [debondings not calculating](https://github.com/bitcat365/oasisscan-backend/blob/f1dafe3/src/main/java/romever/scan/oasisscan/service/AccountService.java#L210)

Reply:
Since Oasisscan is slow about improving this, I added TODO comment. I would merge with this approximation :shrug:

Ready for review